src/adapters/replicantx-staking.adapter.ts

```diff
@@ -0,0 +1,54 @@
+import {BindingScope, extensionFor, injectable} from '@loopback/core';
```

Comment:
Please copy the license headers from other files to here.

Reply:
@raymondfeng